migration/process-migration/src/main/java/io/camunda/migration/process/adapter/MigrationRepositoryIndex.java

```diff
@@ -7,10 +7,11 @@
  */
 package io.camunda.migration.process.adapter;
 
+import io.camunda.webapps.schema.descriptors.errorHandling.ErrorThrowing;
 import io.camunda.webapps.schema.descriptors.operate.OperateIndexDescriptor;
 
 /* Fork of Operate's MigrationRepositoryIndex descriptor for correct prefixing */
-public class MigrationRepositoryIndex extends OperateIndexDescriptor {
+public class MigrationRepositoryIndex extends OperateIndexDescriptor implements ErrorThrowing {
```

Comment:
💭 I'm not sure whether I would expect this on the index classes tbh. I thought we can do this on the handlers.